apps/src/turtle/artist.js

```diff
@@ -341,6 +349,20 @@ Artist.prototype.init = function (config) {
     if (finishButton) {
       dom.addClickTouchEvent(finishButton, this.checkAnswer.bind(this));
     }
+
+    if (this.autoRun) {
```

Comment:
Should any of this live in https://github.com/code-dot-org/artist?  If a 3rd party wanted to create a demo like the on in the .gif what additional code would that require?

Reply:
That repo just has a visualizer, anyone who integrates it still has to provide their own code editor, and then decide when/how often to call [display()](https://github.com/code-dot-org/artist/blob/master/visualization.js#L216). Just like I did in this PR, they would have to start calling `display()` on every code change in their code editor, in addition to/instead of their version of a run button click.